.github/workflows/helm-lint.yaml

```diff
@@ -0,0 +1,27 @@
+name: Helm lint
+# Reusable workflow to lint Helm charts
+
+on:
+  workflow_call:
+    inputs:
+      lint-config-path:
+        description: "The path to the lint configuration file"
+        required: false
+        type: string
+        default: ".github/lint-config.yaml"
+      ref:
+        description: "The ref name to checkout the repository"
+        required: false
+        type: string
+        default: "${{ github.ref_name }}"
+
+jobs:
+  helm-lint:
+    name: Helm Lint
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Lint Helm chart
+        uses: bakdata/ci-templates/actions/helm-lint@feat/lint-default-action
```

Comment:
TODO

Reply:
the other PR needs to be merged first so that i can replace it. Is that what you meant?

Reply:
yep!

Reply:
👍🏾  will do  thanks ☺️

Reply:
since there is no workflow anymore, i think this `todo` can be removed